diff -urNp -x '*.orig' SOAPpy-0.12.22.org/src/SOAPpy/WSDL.py SOAPpy-0.12.22/src/SOAPpy/WSDL.py
--- SOAPpy-0.12.22.org/src/SOAPpy/WSDL.py	2014-06-20 11:25:21.000000000 +0200
+++ SOAPpy-0.12.22/src/SOAPpy/WSDL.py	2021-03-06 21:08:21.808763276 +0100
@@ -81,6 +81,11 @@ class Proxy:
                       (wsdlsource, "\t".join(buf))
             
         if self.wsdl is None:
+            try:
+                self.wsdl = reader.loadFromURL(wsdlsource)
+            except (IOError, OSError): pass
+
+        if self.wsdl is None:
             import StringIO
             self.wsdl = reader.loadFromString(str(wsdlsource))
             #print 'string'
