diff -ur SOAPpy-0.12.0-orig/SOAPpy/Client.py SOAPpy-0.12.0/SOAPpy/Client.py
--- SOAPpy-0.12.0-orig/SOAPpy/Client.py	2005-02-21 21:27:09.000000000 +0100
+++ SOAPpy-0.12.0/SOAPpy/Client.py	2006-11-07 10:36:40.388147000 +0100
@@ -39,12 +39,11 @@
 #
 ################################################################################
 """
+from __future__ import nested_scopes
 
 ident = '$Id$'
 from version import __version__
 
-from __future__ import nested_scopes
-
 #import xml.sax
 import urllib
 from types import *
Tylko w SOAPpy-0.12.0/SOAPpy: Client.pyc
Tylko w SOAPpy-0.12.0/SOAPpy: Config.pyc
Tylko w SOAPpy-0.12.0/SOAPpy: Errors.pyc
Tylko w SOAPpy-0.12.0/SOAPpy: NS.pyc
Tylko w SOAPpy-0.12.0/SOAPpy: Parser.pyc
diff -ur SOAPpy-0.12.0-orig/SOAPpy/Server.py SOAPpy-0.12.0/SOAPpy/Server.py
--- SOAPpy-0.12.0-orig/SOAPpy/Server.py	2005-02-15 17:32:22.000000000 +0100
+++ SOAPpy-0.12.0/SOAPpy/Server.py	2006-11-07 10:36:55.441087750 +0100
@@ -40,11 +40,11 @@
 ################################################################################
 """
 
+from __future__ import nested_scopes
+
 ident = '$Id$'
 from version import __version__
 
-from __future__ import nested_scopes
-
 #import xml.sax
 import re
 import socket
Tylko w SOAPpy-0.12.0/SOAPpy: Server.py~
Tylko w SOAPpy-0.12.0/SOAPpy: SOAPBuilder.pyc
diff -ur SOAPpy-0.12.0-orig/SOAPpy/Types.py SOAPpy-0.12.0/SOAPpy/Types.py
--- SOAPpy-0.12.0-orig/SOAPpy/Types.py	2005-02-22 05:29:43.000000000 +0100
+++ SOAPpy-0.12.0/SOAPpy/Types.py	2006-11-07 10:36:40.392147250 +0100
@@ -33,11 +33,11 @@
 ################################################################################
 """
 
+from __future__ import nested_scopes
+
 ident = '$Id$'
 from version import __version__
 
-from __future__ import nested_scopes
-
 import UserList
 import base64
 import cgi
Tylko w SOAPpy-0.12.0/SOAPpy: Types.pyc
Tylko w SOAPpy-0.12.0/SOAPpy: Utilities.pyc
Tylko w SOAPpy-0.12.0/SOAPpy/wstools: c14n.pyc
Tylko w SOAPpy-0.12.0/SOAPpy/wstools: __init__.pyc
Tylko w SOAPpy-0.12.0/SOAPpy/wstools: logging.pyc
Tylko w SOAPpy-0.12.0/SOAPpy/wstools: Namespaces.pyc
Tylko w SOAPpy-0.12.0/SOAPpy/wstools: TimeoutSocket.pyc
Tylko w SOAPpy-0.12.0/SOAPpy/wstools: Utility.pyc
Tylko w SOAPpy-0.12.0/SOAPpy/wstools: WSDLTools.pyc
Tylko w SOAPpy-0.12.0/SOAPpy/wstools: XMLname.pyc
Tylko w SOAPpy-0.12.0/SOAPpy/wstools: XMLSchema.pyc
